### PR TITLE
libaudiofile: updated source url (v0.3.6)

### PR DESCRIPTION
**Pull request by mchsk**
Description: libaudiofile: updated source url (v0.3.6)

Signed-off-by: Marek Sedlak bodka.zavinac@gmail.com

### CODE REVIEW

**Comment by diizzyy** [comment]
Please squash this into one commit

File: libs/libaudiofile/Makefile [hunk 1]
```diff
@@ -12,9 +12,7 @@ PKG_VERSION:=0.3.6
 PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:= \
-	http://github.com/downloads/mpruett/audiofile/ \
-	http://www.68k.org/~michael/audiofile/
+PKG_SOURCE_URL:=http://audiofile.68k.org/
```
Comment (diizzyy) [inline libs/libaudiofile/Makefile]: What you want is the following?
PKG_SOURCE_URL:=@GNOME/$(PKG_NAME)/0.3

File: libs/libaudiofile/Makefile [hunk 2]
```diff
@@ -12,9 +12,7 @@ PKG_VERSION:=0.3.6
...
 PKG_MD5SUM:=2731d79bec0acef3d30d2fc86b0b72fd
```
Comment (diizzyy) [inline libs/libaudiofile/Makefile]: While at it, please update to SHA256 hash

File: libs/libaudiofile/Makefile (hunk 1 above)
Comment (mchsk) [inline libs/libaudiofile/Makefile]: Hi @diizzyy Thanks for pointing this out! Cleaner solution :)

**Comment by mchsk** [comment]
@diizzyy fixed gnome repo src, squashed, successfully compiled

**Comment by thess** [comment]
Thanks

**Pull request merged**